cft/deploy.yml

```diff
@@ -0,0 +1,38 @@
+Resources:
+  ElasticsearchDomain:
+    Type: AWS::Elasticsearch::Domain
+    Properties:
+      DomainName: "test"
+      ElasticsearchClusterConfig:
+        DedicatedMasterEnabled: true
+        InstanceCount: 2
+        ZoneAwarenessEnabled: true
+        InstanceType: "m3.medium.elasticsearch"
+        DedicatedMasterType: "m3.medium.elasticsearch"
+        DedicatedMasterCount: 3
+      EncryptionAtRestOptions:
+        Enabled: false
+      EBSOptions:
+        EBSEnabled: true
+        Iops: 0
+        VolumeSize: 20
+        VolumeType: "gp2"
+      SnapshotOptions:
+        AutomatedSnapshotStartHour: 0
```

Comment:
Coverity Rapid Scan found issue: Disk-level encryption is disabled - CWE-313, High Severity

**The Amazon ElasticSearch Domain does not have encryption enabled for data at rest.
**

**How to fix:** Enable encryption for data at rest by explicitly setting the `Properties.EncryptionAtRestOptions.Enabled` value to `true`.

```suggestion
        Enabled: true
```